container.go

```diff
@@ -111,6 +124,14 @@ type containerStore interface {
 	// type across all the Scopes that are in effect of this containerStore.
 	getAllValueProviders(name string, t reflect.Type) []provider
 
+	// Returns the decorators that can produce values for the given name and
+	// type.
```

Comment:
nit: for consistency with comment on line 112, since we are iterating over a map, returned providers are probably also undefined? or do we not care about ordering here?

Reply:
The ordering here isn't undefined since it's iterating over a single entry in a map. The value of that map is a list so the ordering will be consistent.